locale/en/download/package-manager.md

```diff
@@ -358,7 +358,7 @@ Using **[MacPorts](http://www.macports.org/)**:
 port install nodejs<major version>
 
 # Example
-port install nodejs7
+port install nodejs8
```

Comment:
Not sure if this is already available. The package is not listed here https://www.macports.org/ports.php?by=name&substr=nodejs